packages/components/atoms/f-image-tile/src/components/ImageTile.vue

```diff
@@ -147,8 +186,25 @@ export default {
   border: 1px solid green;
 }
 
+.c-imageTile-imageContainer {
+  display: block;
+  background-color: rgba($image-tile-background-color, $image-tile-background-opacity);
+  border-radius: $radius-rounded-b;
+  background-image: var(--bg-image);
```

Comment:
I absolutely love the use of css variables here and might retro fit it into the restaurant card! 

I suppose now we don't support IE 11 we're good to use them? Just want to confirm because it's so nice!

Reply:
Totally. I checked [caniuse](https://caniuse.com/css-variables) and support outside IE11 looks pretty good. Where it isn't supported this will fallback to just an orange background. But potentially I could improve this by using @supports.

Reply:
I'm going to leave this as is for now - but create another task/ticket to see if I can improve the experience for places where css variables aren't supported 👍